### PR TITLE
Document reusable workflow limitation

### LINKED ISSUE #11830: Listed limitations of reusable workflows incomplete
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#limitations

### What part(s) of the article would you like to see updated?

I would like to have this limitation included:

> * Reusable workflows do not support the `strategy` property.

That is because I tried using a reusable workflow with that property but GitHub Actions errored with this error:
> The workflow is not valid. .github/workflows/deploy_release.yml (Line: 22, Col: 5): Unexpected value 'strategy' .github/workflows/deploy_release.yml (Line: 31, Col: 21): Unrecognized named-value: 'matrix'. Located at position 1 within expression: matrix.package_type

### Additional information

afbeelding

### PR DESCRIPTION
**Pull request by cmitz**
### Why:

Closes https://github.com/github/docs/issues/11830

I discovered that reusable workflows do not support the `strategy` property and that was not documented anywhere. So, I added that to the list of limitations.

That issue, however, still needs to be confirmed.

### What's being changed:

Very little is being changed. Just one line with a limitation was added to an existing list. 

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@cmitz 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by skedwards88** [comment]
Closing via https://github.com/github/docs/issues/11830#issuecomment-982181729

File: content/actions/learn-github-actions/reusing-workflows.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ Called workflows can access self-hosted runners from caller's context. This mean
 * Reusable workflows stored within a private repository can only be used by workflows within the same repository.
 * Any environment variables set in an `env` context defined at the workflow level in the caller workflow are not propagated to the called workflow. For more information about the `env` context, see "[Context and expression syntax for GitHub Actions](/actions/reference/context-and-expression-syntax-for-github-actions#env-context)."
 * You can't set the concurrency of a called workflow from the caller workflow. For more information about `jobs.<job_id>.concurrency`, see "[Workflow syntax for GitHub Actions](/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idconcurrency)."
+* Reusable workflows do not support the `strategy` property.
```
Comment (skedwards88) [inline content/actions/learn-github-actions/reusing-workflows.md:73]:
```suggestion
* The `strategy` property is not supported in any job that calls a reusable workflow.
```

**Review by skedwards88 — COMMENTED** [review]
Thank you for this PR (and for the clarifying issue)! I added a suggestion to clarify that this just applies to jobs that call another workflow. I'll go ahead an incorporate the suggestion then get this merged down for you.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**